sys/net/gnrc/pktdump/gnrc_pktdump.c

```diff
@@ -101,10 +107,17 @@ static void _dump_snip(gnrc_pktsnip_t *pkt)
             break;
 #endif
         default:
+            (void)hdr_len;  /* in case none of the ifdef'd cases are
+                             * compiled in*/
             printf("NETTYPE_UNKNOWN (%i)\n", pkt->type);
             od_hex_dump(pkt->data, pkt->size, OD_WIDTH_DEFAULT);
             break;
     }
+    if (hdr_len < pkt->size) {
+        size_t size = pkt->size - hdr_len;
+
```

Comment:
maybe add something like `printf("PAYLOAD/DATA\n");` before dumping the rest?

Reply:
Why? It's clearly something else (because before it is not dumping in hex blocks) and also it's not always the payload of the packet (but might be additional data in a header).

Reply:
@smlng I'm with @miri64 here